lusSTR/cli.py

```diff
@@ -28,140 +41,8 @@ def get_parser():
         'input', help='sample(s) in CSV format; first four columns must be Locus, NumReads, '
         'Sequence, SampleID'
     )
```

Comment:
The input and output arguments should be moved out of the master argument parser and handled by each subparser individually.